test/Component/Configuration/Loader/UrlLoaderTest.php

```diff
@@ -34,7 +34,7 @@ public function testGetCodeBlockDisabled()
 
     public function testGetCodeBlock()
     {
-        $config = new UrlLoader(['loaders' => ['url' => ['enabled' => true, 'extensions' => 'png', 'limit' => 100]]]);
+        $config = new UrlLoader(['loaders' => ['url' => ['enabled' => true, 'image_extensions' => 'png', 'font_extensions' => 'woff2', 'limit' => 100]]]);
         $block  = $config->getCodeBlocks()[0];
```

Comment:
Can you add a test case where you test with multiple and no extensions of both? It's mainly because of the different behavior they have now when you've got multiple fonts/images.